modules/ROOT/pages/_partials/supported-versions.adoc

```diff
@@ -43,6 +38,26 @@ ifdef::is-fullpage[== Officially Supported Versions]
 
 Couchbase Lite for C is available on the platforms shown in the tables below.
 
+[IMPORTANT]
```

Comment:
This markup looks great, don't think we lose anything by removing the indirection 👍 
Assume you've tested and the rendered output looks good.

Reply:
Yep it's exactly the same 👍🏼